src/parse/asp/objects.go

```diff
@@ -183,30 +183,43 @@ func (i pyInt) Property(name string) pyObject {
 }
 
 func (i pyInt) Operator(operator Operator, operand pyObject) pyObject {
-	i2, ok := operand.(pyInt)
-	if !ok {
-		panic("Cannot operate on int and " + operand.Type())
-	}
-	switch operator {
-	case Add:
-		return i + i2
-	case Subtract:
-		return i - i2
-	case LessThan:
-		return newPyBool(i < i2)
-	case GreaterThan:
-		return newPyBool(i > i2)
-	case LessThanOrEqual:
-		return newPyBool(i <= i2)
-	case GreaterThanOrEqual:
-		return newPyBool(i >= i2)
-	case Modulo:
-		return i % i2
-	case In:
-		panic("bad operator: 'in' int")
+	switch o := operand.(type) {
+	case pyInt:
+		switch operator {
+		case Add:
+			return i + o
+		case Subtract:
+			return i - o
+		case Multiply:
+			return i * o
+		case Divide:
+			return i / o
+		case LessThan:
+			return newPyBool(i < o)
+		case GreaterThan:
+			return newPyBool(i > o)
+		case LessThanOrEqual:
+			return newPyBool(i <= o)
+		case GreaterThanOrEqual:
+			return newPyBool(i >= o)
+		case Modulo:
+			return i % o
```

Comment:
:trollface: I would've put Modulo right below Divide, but not fussed.

Reply:
Given that the order doesn't matter in a functional sense I'm not going to move this one... :)